decorators/nameable_decorator.rb

```diff
@@ -0,0 +1,3 @@
+class Nameable
+  def correct_name; end
```

Comment:
- [ ] This is a requirement to implement a method called correct_name that will ** raise a NotImplementedError**. Please 🙏🏽  complete the correct_name method to raise a NotImplementedError. We could review this [link](https://refactoring.guru/design-patterns/decorator/ruby/example#example-0) for help.